config/webpack/partial/pwa.js

```diff
@@ -63,7 +63,10 @@ function createEntryConfigFromScripts(importScripts, entry) {
 
 module.exports = function () {
   return function (config) {
-    var swConfig = getSWConfig();
+    var swConfig = assign({
+      cacheId: "electrode",
+      maximumFileSizeToCacheInBytes: 4194304
+    }, getSWConfig());
```

Comment:
I think we want to keep this stuff the way it was since the `sw-config.js` has namespaces these properties under `cache`.  This gives us the whole config with the `cache` and `manifest` config objects, then we use two `assign` below to handle the defaults.